_pages/neurips_2020/neurips-2020-apply.html

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Since we are linking to the CMT site, this page is no longer necessary and can be deleted.